packages/create/package.json

```diff
@@ -51,6 +51,7 @@
     "@chainsafe/libp2p-noise": "^11.0.0",
     "@libp2p/mplex": "^7.1.1",
     "@libp2p/websockets": "^5.0.3",
+    "@waku/relay": "^0.0.1",
```

Comment:
@weboko didn't we say we would keep `*` for packages in the mono repo? I can see the other package have a specified version too.

Reply:
@fryorcraken , right now we have a mix: some are using asterisk and some version. 

Honestly I can't remember on what we settled but it seems to me now we agreed that version will be changed / maintained by `release-please` to specific version instead of asterisk - [last release](https://github.com/waku-org/js-waku/pull/1296/files#diff-41223ec0a874706796f8610dd14e22f489efb5c9819ca962767bc5e92ae6fc78). 

@danisharora099 I believe the task is not needed except if we want to define versions to private packages, I think they won't be changed automatically by the release CI. That can be done manually it seems.